site/templates/base.html

```diff
@@ -2,11 +2,12 @@
   <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
     <meta http-equiv="content-language" content="en" />
-    <meta name="description" content="" />
-    <meta name="keywords" content="" />
+    <meta name="description" content="ONO digital security Game by Tactical Tech Collective!" />
+    <link href="/static/images/favicon.ico" rel="icon" type="image/x-icon">
     <title>{% block title %}{% endblock %} Ono - The Game</title>
     <link rel="stylesheet" href="/static/css/ono.css" type="text/css" media="screen" charset="utf-8" />
     <link rel="stylesheet" href="/static/css/animate.css" type="text/css" media="screen" charset="utf-8" />
+    <script src="static/js/jquery-1.11.2.min.js"></script>
```

Comment:
we don't use jquery, why add this? If we use js, then it should only be used in the particular file that needs it, not in the base.html. it would be best to avoid js altogether if possible.